### PR TITLE
Updates RNs for 4.10.53 z-stream release

### PR DESCRIPTION
**Pull request by cbippley**
Version(s):
4.10

Issue:
[OSDOCS-5409](https://issues.redhat.com/browse/OSDOCS-5409)

Link to docs preview:
[Preview](http://file.rdu.redhat.com/~cbippley/RNs-4.10.53/release_notes/ocp-4-10-release-notes.html)
QE review:
- [ ] QE has approved this change.

### CODE REVIEW

**Comment by mburke5678** [comment]
@cbippley  A few nits. Othwerwise LGTM

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -3837,4 +3837,31 @@ $ oc adm release info 4.10.52 --pullspecs
...
+
+[source,terminal]
+----
+$ oc adm release info 4.10.53 --pullspecs
+----
+
+[id="ocp-4-10-53-bug-fixes"]
+==== Bug fixes
+
+* In order to be compatible with OpenStack clouds that don't have Swift installed, cluster-image-registry-operator has a mechanism for automatically choosing the storage back-end during the first boot. If Swift is available, Swift is used. Otherwise, a persistent volume claim (PVC) is issued and block storage is used. Previously, cluster-image-registry-operator (CIRO) would fall back to using a PVC when it failed to reach Swift. In particular, lack of connectivity during the first boot would make CIRO fall back to using a PVC. With this change, failure to reach the OpenStack API, or other incidental failures, cause CIRO to retry the probe. The fallback to PVC will only occur if the OpenStack catalog is correctly found, and it does not contain object storage, or if the current user does not have permission to list containers. (link:https://issues.redhat.com/browse/OCPBUGS-5974[*OCPBUGS-5974*])
```
[the remark this answers is not in the document]
Comment (cbippley) [reply release_notes/ocp-4-10-release-notes.adoc:3859]: Thank you!

**Pull request merged**